packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -269,8 +272,11 @@ const AnalyticalTable: FC<TableProps> = forwardRef((props: TableProps, ref: Ref<
 
   useEffect(() => {
     toggleAllRowsSelected(false);
+    const validChars = /^[0-9.]+$/;
     for (const row in selectedRowIds) {
-      toggleRowSelected(row, selectedRowIds[row]);
+      if (row.match(validChars)) {
```

Comment:
`RegExp.test` return a `boolean`, so that would be the better fit in my opinion

```suggestion
      if (row.test(validChars)) {
```